### PR TITLE
Fix flappers

### PR DESCRIPTION
**Pull request by derekcollison**
Signed-off-by: Derek Collison <derek@nats.io>

/cc @nats-io/core

### CODE REVIEW

File: server/jetstream_cluster_test.go [hunk 1]
```diff
@@ -5386,7 +5389,7 @@ func TestJetStreamClusterAckPendingWithMaxRedelivered(t *testing.T) {
 	}
 
 	// Send in 100 messages.
-	msg, toSend := make([]byte, 32), 100
+	msg, toSend := []byte("HELLO"), 100
 	rand.Read(msg)
```
Comment (kozlovic) [inline server/jetstream_cluster_test.go:5393]: Is that necessary?

**Review by kozlovic — APPROVED** [review]
LGTM. Just use of rand.Read(msg) that does not seem to be required.

File: server/jetstream_cluster_test.go (hunk 1 above)
Comment (derekcollison) [reply server/jetstream_cluster_test.go:5393]: Not as it is but when I was debugging and printing things out it was much easier.

Comment (derekcollison) [reply server/jetstream_cluster_test.go:5393]: Does not affect the test.

**Pull request merged**